components/checks/passive/common_admin_interfaces.rb

```diff
@@ -0,0 +1,56 @@
+=begin
+    Copyright 2010-2014 Tasos Laskos <tasos.laskos@arachni-scanner.com>
+
+    This file is part of the Arachni Framework project and is subject to
+    redistribution and commercial restrictions. Please see the Arachni Framework
+    web site for more information on licensing and terms of use.
+=end
+
+# Looks for common administration interfaces on the server.
+#
+# @author Brendan Coles <bcoles@gmail.com>
+#
+# @version 0.1
+class Arachni::Checks::CommonAdminInterfaces < Arachni::Check::Base
+
+    def self.filenames
+        @filenames ||= read_file( 'admin-panels.txt' ) # wfuzz - wordlist/general/admin-panels.txt
```

Comment:
Hmm you're right. I doubt they'll be keen given Arachni's dual license.

Instead, I've updated the admin-panels.txt file. This file now contains a list of common admin directories in English and a few Spanish and French translations.

Alternatively, [AdminpageFinder.py](http://dl.packetstormsecurity.net/UNIX/utilities/AdminpageFinder.py.txt) does not appear to be licensed and has a long list of admin panels.